source/nodejs/package-lock.json

```diff
@@ -497,14 +497,14 @@
       }
     },
     "@lerna/add": {
-      "version": "3.13.3",
-      "resolved": "https://registry.npmjs.org/@lerna/add/-/add-3.13.3.tgz",
-      "integrity": "sha512-T3/Lsbo9ZFq+vL3ssaHxA8oKikZAPTJTGFe4CRuQgWCDd/M61+51jeWsngdaHpwzSSRDRjxg8fJTG10y10pnfA==",
+      "version": "3.14.0",
```

Comment:
Curious -- upgrades to these packages driven by functionality need or security flagging or just because?

Reply:
These get updated as you compile. I have nothing to do with this change, and never understood why/when they happen.